vale/styles/jargonLint/Licenses.yml

```diff
@@ -0,0 +1,41 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  Apache[ -]?2(?:\.0)?:Apache 2.0
+  [0O]BSD: 0BSD
+  BSD: BSD
+  BSD ?1: BSD-1
+  BSD ?2: BSD-2
+  BSD ?3: BSD-3
```

Comment:
```suggestion
  BSD[ -]?3: BSD-3
```

Reply:
I found tons of

BSD 3-clause

https://github.com/search?q=language%3Amarkdown+%22BSD+3-clause%22+&type=code